### PR TITLE
add vmdk job

### PR DESCRIPTION
**Pull request by yih-redhat**
Add ostree-vsphere.sh in rhel9

### CODE REVIEW

**Review by henrywang — CHANGES_REQUESTED** [review]

File: .github/workflows/rhel-9.yml [hunk 1]
```diff
@@ -763,3 +763,108 @@ jobs:
...
+        run: schutzbot/deploy.sh
+        env:
+          GIT_COMMIT: ${{ needs.pr-info.outputs.sha }}
+        timeout-minutes: 20
+
+      - name: run ostree-vsphere.sh
+        run: /usr/libexec/tests/osbuild-composer/ostree-vsphere.sh
+        env:
+          DOWNLOAD_NODE: ${{ secrets.DOWNLOAD_NODE }}
+        timeout-minutes: 60
```
Comment (henrywang) [inline .github/workflows/rhel-9.yml:838]: Is 60 minutes enough for your test?

**Comment by henrywang** [comment]
@yih-redhat Could you please add job for CS9 as well? Thanks.

File: .github/workflows/rhel-9.yml [hunk 2]
```diff
@@ -763,3 +763,108 @@ jobs:
...
+      - name: Make a deployment
+        run: schutzbot/deploy.sh
+        env:
+          GIT_COMMIT: ${{ needs.pr-info.outputs.sha }}
+        timeout-minutes: 20
+
+      - name: run ostree-vsphere.sh
+        run: /usr/libexec/tests/osbuild-composer/ostree-vsphere.sh
+        env:
+          DOWNLOAD_NODE: ${{ secrets.DOWNLOAD_NODE }}
```
[the remark this answers is not in the document]
Comment (yih-redhat) [reply .github/workflows/rhel-9.yml:837]: Ok, just one thing, the password contains !, and need single quote to escape it, let's see if it works here.

File: .github/workflows/rhel-9.yml (hunk 1 above)
Comment (yih-redhat) [reply .github/workflows/rhel-9.yml:838]: 60 is not enough, 120 minutes should be ok.

File: .github/workflows/rhel-9.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (yih-redhat) [reply .github/workflows/rhel-9.yml:837]: fixed.

File: .github/workflows/rhel-9.yml (hunk 1 above)
Comment (yih-redhat) [reply .github/workflows/rhel-9.yml]: changed to 120

**Comment by yih-redhat** [comment]
> @yih-redhat Could you please add job for CS9 as well? Thanks.

Added ostree vsphere job in cs9.

**Review by henrywang — CHANGES_REQUESTED** [review]

File: .github/workflows/rhel-9.yml [hunk 3]
```diff
@@ -763,3 +763,112 @@ jobs:
...
+          context: ${{ env.STATUS_NAME }}
+          state: pending
+          description: 'RHEL 9.3 - Runner has been deploying...'
+          target_url: '${{ steps.job-log-url.outputs.html_url }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+
+  edge-vsphere-9:
+    needs: [pr-info, pre-edge-vsphere-9]
+    runs-on: [kite, x86_64, rhos-0x, rhel-9-3, large]
```
Comment (henrywang) [inline .github/workflows/rhel-9.yml:795]: Please change the instance type to medium

Comment (yih-redhat) [reply .github/workflows/rhel-9.yml]: done.

File: .github/workflows/cs9.yml [hunk 4]
```diff
@@ -762,3 +762,112 @@ jobs:
...
+          context: ${{ env.STATUS_NAME }}
+          state: pending
+          description: 'CentOS Stream 9 - Runner has been deploying...'
+          target_url: '${{ steps.job-log-url.outputs.html_url }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT }}
+
+  edge-vsphere-cs9:
+    needs: [pr-info, pre-edge-vsphere-cs9]
+    runs-on: [kite, x86_64, gcp, centos-stream-9, medium]
```
[the remark this answers is not in the document]
Comment (yih-redhat) [reply .github/workflows/cs9.yml]: done.

**Review by henrywang — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**